app/src/common/shared/com/igalia/wolvic/browser/api/WSession.java

```diff
@@ -2580,10 +2580,71 @@ default void goForward() {
     @NonNull
     String getDefaultUserAgent(final int mode);
 
+    @AnyThread
+    @NonNull
+    WSession.SessionFinder getFinder();
```

Comment:
Bikeshedding a bit but "finder" is way too generic. Let's use `getSessionFinder` instead